content/blog/new-kubernetes-superpowers/index.md

```diff
@@ -291,7 +291,7 @@ func main() {
 
 ## OPA support in Pulumi CrossGuard Policy-as-Code
 
-Pulumi’s [CrossGuard](https://www.pulumi.com/docs/guides/crossguard/) policy-as-code framework enforces policy across all your modern cloud infrastructure and/or in Kubernetes - using the full expressiveness of rich programming languages. It is deeply integrated into the deployment process to ensure that policy violations are identified *before* infrastructure is deployed instead of minutes or hours after changes are made.
+Pulumi’s [CrossGuard]({{< relref “/docs/guides/crossguard” >}}) policy-as-code framework enforces policy across all your modern cloud infrastructure and/or in Kubernetes - using the full expressiveness of rich programming languages. It is deeply integrated into the deployment process to ensure that policy violations are identified *before* infrastructure is deployed instead of minutes or hours after changes are made.
```

Comment:
```suggestion
Pulumi’s [CrossGuard]({{< relref "/docs/guides/crossguard" >}}) policy-as-code framework enforces policy across all your modern cloud infrastructure and/or in Kubernetes - using the full expressiveness of rich programming languages. It is deeply integrated into the deployment process to ensure that policy violations are identified *before* infrastructure is deployed instead of minutes or hours after changes are made.
```